src/aioamazondevices/api.py

```diff
@@ -763,6 +783,128 @@ async def _response_to_json(self, raw_resp: ClientResponse) -> dict[str, Any]:
         except orjson.JSONDecodeError as exc:
             raise ValueError("Response with corrupted JSON format") from exc
 
+    async def _get_notifications(self) -> dict[str, dict[str, AmazonSchedule]]:
+        final_notifications: dict[str, dict[str, AmazonSchedule]] = {}
+
+        _, raw_resp = await self._session_request(
+            HTTPMethod.GET, url=f"https://alexa.amazon.{self._domain}/api/notifications"
+        )
+        notifications = await self._response_to_json(raw_resp)
+        for schedule in notifications["notifications"]:
+            schedule_type: str = schedule["type"]
+            schedule_device_serial = schedule["deviceSerialNumber"]
+            label_desc = schedule_type.lower() + "Label"
```

Comment:
```suggestion
            if schedule_type in NOTIFICATION_ALARM:
                label_desc = "alarmLabel"
            else:
                label_desc = schedule_type.lower() + "Label"
```
Includes music alarms.